File: 프로그래머스/lv0/120826. 특정 문자 제거하기/특정 문자 제거하기.java
class Solution {
    public String solution(String my_string, String letter) {
    	String answer = "";
    	String[] strArr = my_string.split("");
    	
    	for(int i=0; i<strArr.length;i++) {
    		if(!strArr[i].equals(letter)) {
    			answer=answer+strArr[i];
    		}
    	}
    	 return answer;
    }
}

File: 프로그래머스/lv0/120826. 특정 문자 제거하기/특정 문자 제거하기.cs
using System;

public class Solution {
    public string solution(string my_string, string letter) {
        string answer = "";
        string[] tmp = my_string.Split(letter);
        
        for(int i=0; i<tmp.Length; i++){
            answer += tmp[i];
        }
        
        return answer;
    }
}